projects.go

```diff
@@ -678,18 +678,28 @@ type CreateProjectOptions struct {
 	Visibility                                *VisibilityValue                     `url:"visibility,omitempty" json:"visibility,omitempty"`
 	WikiAccessLevel                           *AccessControlValue                  `url:"wiki_access_level,omitempty" json:"wiki_access_level,omitempty"`
 
-	// Deprecated members
-	CIForwardDeploymentEnabled *bool     `url:"ci_forward_deployment_enabled,omitempty" json:"ci_forward_deployment_enabled,omitempty"`
-	ContainerRegistryEnabled   *bool     `url:"container_registry_enabled,omitempty" json:"container_registry_enabled,omitempty"`
-	IssuesEnabled              *bool     `url:"issues_enabled,omitempty" json:"issues_enabled,omitempty"`
-	IssuesTemplate             *string   `url:"issues_template,omitempty" json:"issues_template,omitempty"`
-	JobsEnabled                *bool     `url:"jobs_enabled,omitempty" json:"jobs_enabled,omitempty"`
-	MergeRequestsEnabled       *bool     `url:"merge_requests_enabled,omitempty" json:"merge_requests_enabled,omitempty"`
-	MergeRequestsTemplate      *string   `url:"merge_requests_template,omitempty" json:"merge_requests_template,omitempty"`
-	ServiceDeskEnabled         *bool     `url:"service_desk_enabled,omitempty" json:"service_desk_enabled,omitempty"`
-	SnippetsEnabled            *bool     `url:"snippets_enabled,omitempty" json:"snippets_enabled,omitempty"`
-	TagList                    *[]string `url:"tag_list,omitempty" json:"tag_list,omitempty"`
-	WikiEnabled                *bool     `url:"wiki_enabled,omitempty" json:"wiki_enabled,omitempty"`
+	// Deprecated: Unknown reason.
+	CIForwardDeploymentEnabled *bool `url:"ci_forward_deployment_enabled,omitempty" json:"ci_forward_deployment_enabled,omitempty"`
```

Comment:
Should this parameter even exist?

Reply:
My guess is this is from a very old version... I'm pretty sure nothing was ever added without having checked for its existence. So I guess for all the fields you marked with `Unknown reason` we should say something like `No longer supported in recent versions` unless we are willing to invest the time to find out when it was last used...

Reply:
Unfortunately, it is hard to prove that something never existed. Therefore, I will apply your suggestion.

[GitLab issue #341672: Allow setting ci_forward_deployment_enabled at project creation](https://gitlab.com/gitlab-org/gitlab/-/issues/341672)